TestStack.Dossier.Tests/TestHelpers/StaticAnonymousValueSupplier.cs

```diff
@@ -14,9 +16,19 @@ public StaticAnonymousValueSupplier(object valueToSupply)
             return typeof(TValue) == _valueToSupply.GetType();
         }
 
+        public bool CanSupplyValue(Type type, string propertyName)
```

Comment:
Do we need the generic version anymore? If not can we delete them so there isn't so much duplicate logic?

That would be a braking change in the public API and require a v3 release (I'm comfortable with that though)